distribution/docs/src/main/resources/_contents/_configuring/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -51,14 +50,6 @@ The following table shows the format to use for each property type supported.
 |name = L"value"
 |diameter = L"10000"
 
```

Comment:
I pushed an update to add back these examples but with the IEEE 754 format and that the `.config` files support floats and doubles.